Implementation Local available disk space #23

ridvan-sevimli opened the pull request.

Commit:
Implementation Local available disk space #23

Commit:
Implementation Local available disk space #23

ridvan-sevimli commented:
Konnte für dieses feature keinen Test schreiben.

ridvan-sevimli committed:
Aufteilung Controller und Views und Hiflsklassen (#31)

* Aufteilung Controller und Views und Hifls Klassen

* Deletet Controller class

Co-authored-by: ridvan <sevimrid@students.zhaw.ch>

siebl commented:
ev. Kommentar in `FileStorage`, dass grössen in GB erwartet werden oder umbenennen der methode zu `getAvailableStorageSpaceGB` um Verwirrung zu Einheiten vorzubeugen?

Sonst siehts gut aus 👍

ridvan-sevimli commented:
> ev. Kommentar in `FileStorage`, dass grössen in GB erwartet werden oder umbenennen der methode zu `getAvailableStorageSpaceGB` um Verwirrung zu Einheiten vorzubeugen?
> 
> Sonst siehts gut aus 👍

Stimmt guete punkt dank dir 👍🏼 mir gfallt das mit java.io.File net e so mir hänt intern au e Klass wo File isch weiss noni genau öb mir das sötte im Team mol aluege

SandroGuerotto committed:
Feature/19 dropbox and Feature/22 register screen (#35)

* #19 created dropbox connections
added json read/writer
#22 created register form

* updated screens to controlled screens

* #19 implemented test class
added json read/writer tests
#22 updated signup form to controlled screen

ridvan-sevimli committed:
Changed naming convention of classes to avoid conflicts with java standard library (#37)

Co-authored-by: sevimrid <sevimrid@students.zhaw.ch>

Commit:
refactoring java.io.File to File implemented

Commit:
Merge remote-tracking branch 'origin/feature/23-getAvailableStorageSpace' into feature/23-getAvailableStorageSpace

# Conflicts:
#	src/main/java/ch/psc/domain/storage/service/DropBoxService.java
#	src/main/java/ch/psc/domain/storage/service/FileStorage.java
#	src/main/java/ch/psc/domain/storage/service/LocalStorage.java

Commit:
changed comments in to english

siebl reviewed (approved):
Sieht gut aus 👍

SandroGuerotto reviewed (approved):
LGM
